src/Containers/Microsoft.NET.Build.Containers/ManifestV2.cs

```diff
@@ -47,9 +50,9 @@ namespace Microsoft.NET.Build.Containers;
     /// <summary>
     /// Gets the digest for this manifest.
     /// </summary>
-    public string GetDigest() => DigestUtils.GetDigest(JsonSerializer.SerializeToNode(this)?.ToJsonString() ?? string.Empty);
+    public string GetDigest() => KnownDigest ??= DigestUtils.GetDigest(JsonSerializer.SerializeToNode(this)?.ToJsonString(new JsonSerializerOptions() { WriteIndented = true }).ReplaceLineEndings("\r") ?? string.Empty);
```

Comment:
I'm going to roll back the serialization changes here, because they aren't enough to get us to parity with the json from MCR.